README.md

```diff
@@ -151,6 +151,7 @@ These are the VS Code [settings] available for the Extension:
 | `dvc.dvcPath`                          | Path or shell command to the DVC binary. Required unless Microsoft's [Python extension] is installed and the `dvc` package found in its environment.     |
 | `dvc.pythonPath`                       | Path to the desired Python interpreter to use with DVC. Should only be utilized when using a virtual environment without Microsoft's [Python extension]. |
 | `dvc.experimentsTableHeadMaxHeight`    | Maximum height of experiment table head rows.                                                                                                            |
+| `dvc.focusedProjects`                  | A subset of paths to the workspace's available DVC projects. Using this option will override project auto-discovery.                                     |
```

Comment:
how about `dvc.projects`? I'm still not sure about this "focused" term ...

Reply:
IMO projects needs some kind of determiner to signal that it is only needed under some special condition. I considered override, included and current. I also worked through lots of different combinations of this text. I tried to consider the end user.